AirCasting/SessionsSynchronization/Database/DatabaseMeasurement.swift

```diff
@@ -0,0 +1,50 @@
+import Foundation
+
+extension Database {
+    public struct Measurement {
+        public let id: Int
+        public let sensorName: String
+        public let sensorPackageName: String
+        public let unitName: String
+        public let measurementType: String
+        public let measurementShortType: String
+        public let unitSymbol: String
+        public let thresholdVeryLow: Int
+        public let thresholdLow: Int
+        public let thresholdMedium: Int
+        public let thresholdHigh: Int
+        public let thresholdVeryHigh: Int
+        public let isDeleted: Bool
+        
+        #warning("Make this private as soon as we move PersistenceController into here")
```

Comment:
Don't quite get this comment. Acutely at all

Reply:
Good catch, it's a leftover nonsense, removed